BootloaderCorePkg/Tools/SingleSign.py

```diff
@@ -49,9 +49,10 @@
 
 
 def print_message_sbl_key_dir ():
-    print ("Pre-requiste: SBL_KEY_DIR environment variable has to be set!")
-    print ("SBL_KEY_DIR is path to keys used for the project!")
-    print ("For Keys generation follow GenerateKeys.py available in tool directory!")
+    print ("Pre-requiste: SBL_KEY_DIR environment variable has to be set!!")
+    print ("SBL_KEY_DIR is path to keys used for the project!!")
+    print ("TO GENERATE KEYS USE GenerateKeys.py AVAIABLE IN BootloaderCorePkg/Tools DIRECTORY !!")
+    print ("Run python BootloaderCorePkg/Tools/GenerateKeys.py -k $SBL_KEY_DIR!!")

```

Comment:
we have duplicated info for key generate ?  Can we just have one place for this msg ?